telegraf/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telegraf",
-  "versions": ["1.4", "1.5", "1.6"],
+  "versions": ["1.4", "1.5", "1.6", "1.7"],
```

Comment:
Can you erase 1.4 from this and also the 1.4 directory? There aren't going to be anymore telegraf 1.4 releases anyway.

Reply:
Removed